docs/datatypes/contextual/index.md

```diff
@@ -341,17 +342,22 @@ case class LoggingLive(console: Console.Service, clock: Clock.Service) extends L
 4. **Defining ZLayer** — Now, we create a companion object for `LoggingLive` data type and lift the service implementation into the `ZLayer`:
```

Comment:
We should also change this instruction to something like this:

Now, we create a companion object for `Logging` service interface and lift the service implementation into the `ZLayer`:

Reply:
Great point! Updated now.

Reply:
One other change remains under [_Using provideCustomLayer Method_](https://zio.dev/docs/datatypes/contextual/#dependency-propagation) section.

Reply:
Thanks!